gulpfile.js

```diff
@@ -29,23 +29,23 @@ gulp.task('third_party', function() {
       .pipe(gulp.dest('deploy/client/third_party/jquery'));
 
   /** Angular */
-  gulp.src('bower_components/angular/angular.min.*')
+  gulp.src('bower_components/angular/angular.*')
```

Comment:
Gulp has a recipe for dealing with minified vs. non-minified files; this should work well in this case.

https://github.com/gulpjs/gulp/commit/61b5751f46749d3ec4611bb3bf8d07de4faf4f65